### ISSUE TITLE
Header anomaly and content hidden when change from mobile size to desktop size and nav menu is opened

### ISSUE THREAD

**Issue by joseantanaper** [onwidget/astrowind#262]
# Browser: Chrome, Edge

1. Change browser size to a mobile size (then the "Header Navigation Menu" is collapsed).
2. Click on the "Navigation Menu Button" to open navigation links (and DON'T select any link to remain it opened).
3. Change browser size to a desktop size.
4. Then, "Header Navigation Menu" is shown vertically middle centered, and no content is shown. It seems that mobile opened menu remains some garbage here.
5. If click on body (BUT not on original header top position), everything returns to normal ("Header Navigation Menu" is on top and content is shown).

![image](https://github.com/onwidget/astrowind/assets/56954559/1b023ea7-1132-4b0a-b8e8-e9e6f9b42203)

# Expected:

When change to Desktop Size the "Header Navigation Menu" should be at top position and the content should be shown as normal.

**Comment by prototypa** [comment]
Hello @joseantanaper 

Thanks for reporting it. I know about that problem. However, it is something that does not manifest itself easy in the real environment. It usually only happens if the screen size is modified with the menu open on mobile, which usually happens more to developers doing tests than to normal users. For that reason I have deprioritized it and I detected it about 3 months ago.

In any case, it would be good to solve it if it doesn't consume many resources in the attempt. Something simple could be to react to the window's on resize event. But I don't know if this is putting more processing resources into something so simple and possibly no one who uses that normally notices it. Another solution could be to refactor all that code and eliminate the possibility of that problem.

If I have time these days I will give you a solution. If you come up with something, welcome to add a PR.

Thank you.
